resources/views/sellers/index.blade.php

```diff
@@ -2,87 +2,101 @@
+                            <input class="rounded col-12 py-2 px-3" value="Emily" readonly>
+                        </div>
+                        <div class="col-6">
+                            <p class="mb-0">Last Name</p>
+                            <input class="rounded col-12 py-2 px-3" value="Jones" readonly>
+                        </div>
                     </div>
-                    <div class="col-6">
-                        <p class="mb-0">Last Name</p>
-                        <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">Jones</p>
+
+                    <div class="col-12">
+                        <p class="mb-0">Username</p>
+                        <input class="rounded col-12 py-2 px-3" value="EmilyJones22" readonly>
                     </div>
-                </div>
 
-                <div class="col-12">
-                    <p class="mb-0">Username</p>
-                    <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">EmilyJones22</p>
-                </div>
+                    <div class="col-12">
+                        <p class="mb-0">Company Name</p>
+                        <input class="rounded col-12 py-2 px-3" value="NovaLink Global" readonly>
+                    </div>
 
-                <div class="col-12">
-                    <p class="mb-0">Email</p>
-                    <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">emily-jones58@gmail.com</p>
-                </div>
+                    <div class="col-12">
+                        <p class="mb-0">Email</p>
+                        <input class="rounded col-12 py-2 px-3" value="emily-jones58@gmail.com" readonly>
+                    </div>
 
-                <div class="col-12">
-                    <p class="mb-0">Phone Number</p>
-                    <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">(603) 555-0123</p>
+                    <div class="col-12">
+                        <p class="mb-0">Phone Number</p>
+                        <input class="rounded col-12 py-2 px-3" value="(603) 555-0123" readonly>
+                    </div>
                 </div>
-            </div>
 
-            <div class="col-4">
-                <img src="#" alt="" class="rounded-circle picture-circle border border-primary">
+                <div class="col-4">
+                    <img src="#" alt="" class="rounded-circle picture-circle border border-primary">
+                </div>
             </div>
-        </div>
 
-        <div class="row">
-            <div class="col-12">
-                <p class="mb-0">Street Address</p>
-                <p class="rounded py-2 px-3" style="background-color: #E6E6E6;">4140 Parl</p>
-            </div>
+            <div class="row">
+                <div class="col-12">
+                    <p class="mb-0">Street Address</p>
+                    <input class="rounded py-2 px-3" value="4140 Parl" readonly>
```

Comment:
add type attribute
<input type="text"

check also other inputs

Reply:
I fixed the word with [da95828](https://github.com/kredo-school/16th_rankshty/pull/49/commits/da958284dccea7624d70416e87d40bbca078e805) and [7540421](https://github.com/kredo-school/16th_rankshty/pull/49/commits/754042125237aab492ea60a3928db9d581a2c4c8).